Library/Formula/devil.rb

```diff
@@ -1,11 +1,46 @@
 class Devil < Formula
   desc "Cross-platform image library"
   homepage "https://sourceforge.net/projects/openil/"
-  url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/DevIL-1.7.8.tar.gz"
-  sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
   revision 1
 
-  bottle :disable, "Can't generate bottles until builds with either Clang or GCC-5"
+  PATCH_DATA = <<-EOS.undent
+    --- a/src-ILU/ilur/ilur.c 2009-03-08 08:10:12.000000000 +0100
+    +++ b/src-ILU/ilur/ilur.c 2010-09-26 20:01:45.000000000 +0200
+    @@ -1,6 +1,7 @@
+     #include <string.h>
+     #include <stdio.h>
+    -#include <malloc.h>
+    +#include <stdlib.h>
+    +#include "sys/malloc.h"
+
+     #include <IL/il.h>
+     #include <IL/ilu.h>
+  EOS
+
+  stable do
+    url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/" \
+      "DevIL-1.7.8.tar.gz"
+
+    sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
+
+    # fix compilation issue for ilur.c
+    patch PATCH_DATA
+  end
+
+  head do
+    url "https://github.com/DentonW/DevIL.git"
+
+    depends_on "libtool" => :build
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "homebrew/x11/freeglut"
```

Comment:
Yup, that's the one. Looking at the upstream build scripts it looks like it uses it if found rather than by default, so we could probably get away with it being `=> :optional`.